### DISCUSSION TITLE
Doubt to use makefile in STL

### DISCUSSION THREAD

**Discussion by jeet0474** [microsoft/STL#4823] [Ideas] [answered]
I recently built a project and in that I used makefile, in that I only had to make one cpp file and then making custom header files, then put them into main cpp file. From that you have to use only one time main header files you don't need to add main header files in every other file again and again. and you can also see which header files you are using and which are not.
My dout is that can we use makefiles in STL/stl/src/

**Comment by StephanTLavavej** [comment, marked as answer]
Our project uses CMake and Ninja to build its separately compiled DLLs and LIBs.
